docs/static/reserved-fields.asciidoc

```diff
@@ -0,0 +1,39 @@
+[float]
+[[reserved-fields]] 
+==== Reserved fields in {ls} events 
+
+Some fields in {ls} events are reserved, or are required to adhere to a
+certain shape. Avoid using these fields. Otherwise, you can experience runtime
```

Comment:
🤔 I'm not sure we should advise people to "Avoid using these fields" as a blanket statement -- after all we encourage the use of `@timestamp` and `@metadata` in other docs.